Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/venv/\r\n/YOLO Object Detection with OpenCV/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitignore	(revision 301273d654c9c5cc0cd1479453feaf46acefe00d)
+++ .gitignore	(date 1590164619706)
@@ -1,2 +1,4 @@
 /venv/
 /YOLO Object Detection with OpenCV/
+/Recycle_venv/
+/Mobile App/Mobile_App_Venv/
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"Python 3.7 (Recycle)\" project-jdk-type=\"Python SDK\" />\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/misc.xml	(revision 301273d654c9c5cc0cd1479453feaf46acefe00d)
+++ .idea/misc.xml	(date 1590170006932)
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.7 (Recycle)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.7" project-jdk-type="Python SDK" />
 </project>
\ No newline at end of file
Index: .idea/Recycle.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"PYTHON_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <sourceFolder url=\"file://$MODULE_DIR$/OpenCV\" isTestSource=\"false\" />\r\n      <excludeFolder url=\"file://$MODULE_DIR$/venv\" />\r\n    </content>\r\n    <orderEntry type=\"jdk\" jdkName=\"Python 3.7 (Recycle)\" jdkType=\"Python SDK\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/Recycle.iml	(revision 301273d654c9c5cc0cd1479453feaf46acefe00d)
+++ .idea/Recycle.iml	(date 1590170006928)
@@ -2,10 +2,11 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/OpenCV" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/Mobile App/Mobile_App_Venv" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/Mobile App/kivy_venv" isTestSource="false" />
       <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
-    <orderEntry type="jdk" jdkName="Python 3.7 (Recycle)" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.7" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
 </module>
\ No newline at end of file
Index: Mobile App/calculator.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Mobile App/calculator.py	(date 1590172016158)
+++ Mobile App/calculator.py	(date 1590172016158)
@@ -0,0 +1,70 @@
+from kivy.app import App
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.button import Button
+from kivy.uix.textinput import TextInput
+
+class MainApp(App):
+    def build(self):
+        self.operators = ["/", "*", "+", "-"]
+        self.last_was_operator = None
+        self.last_button = None
+        main_layout = BoxLayout(orientation="vertical")
+        self.solution = TextInput(
+            multiline=False, readonly=True, halign="right", font_size=55
+        )
+        main_layout.add_widget(self.solution)
+        buttons = [
+            ["7", "8", "9", "/"],
+            ["4", "5", "6", "*"],
+            ["1", "2", "3", "-"],
+            [".", "0", "C", "+"],
+        ]
+        for row in buttons:
+            h_layout = BoxLayout()
+            for label in row:
+                button = Button(
+                    text=label,
+                    pos_hint={"center_x": 0.5, "center_y": 0.5},
+                )
+                button.bind(on_press=self.on_button_press)
+                h_layout.add_widget(button)
+            main_layout.add_widget(h_layout)
+
+        equals_button = Button(
+            text="=", pos_hint={"center_x": 0.5, "center_y": 0.5}
+        )
+        equals_button.bind(on_press=self.on_solution)
+        main_layout.add_widget(equals_button)
+
+        return main_layout
+
+    def on_button_press(self, instance):
+        current = self.solution.text
+        button_text = instance.text
+
+        if button_text == "C":
+            # Clear the solution widget
+            self.solution.text = ""
+        else:
+            if current and (
+                    self.last_was_operator and button_text in self.operators):
+                # Don't add two operators right after each other
+                return
+            elif current == "" and button_text in self.operators:
+                # First character cannot be an operator
+                return
+            else:
+                new_text = current + button_text
+                self.solution.text = new_text
+        self.last_button = button_text
+        self.last_was_operator = self.last_button in self.operators
+
+    def on_solution(self, instance):
+        text = self.solution.text
+        if text:
+            solution = str(eval(self.solution.text))
+            self.solution.text = solution
+
+if __name__ == "__main__":
+    app = MainApp()
+    app.run()
\ No newline at end of file
Index: Mobile App/KV_language.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Mobile App/KV_language.py	(date 1590171489623)
+++ Mobile App/KV_language.py	(date 1590171489623)
@@ -0,0 +1,17 @@
+from kivy.app import App
+from kivy.uix.button import Button
+
+class ButtonApp(App):
+    def build(self):
+        text: 'Press me'
+        size_hint: (.5, .5)
+        pos_hint: {'center_x': .5, 'center_y': .5}
+        on_press: app.on_press_button()
+        return Button()
+
+    def on_press_button(self):
+        print('You pressed the button!')
+
+if __name__ == '__main__':
+    app = ButtonApp()
+    app.run()
\ No newline at end of file
Index: Mobile App/Hello_Kivy.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Mobile App/Hello_Kivy.py	(date 1590170548471)
+++ Mobile App/Hello_Kivy.py	(date 1590170548471)
@@ -0,0 +1,22 @@
+import kivy
+from kivy.app import App
+from kivy.uix.image import Image
+from kivy.uix.label import Label
+
+class MainApp(App):
+    def build(self):
+        # label = Label(text='Hello from Kivy',
+        #               size_hint=(.5, .5),
+        #               pos_hint={'center_x': .5, 'center_y': .5})
+
+
+
+        img = Image(source='../elephant_gray.jpeg',
+                size_hint=(1, .5),
+                pos_hint={'center_x': .5, 'center_y': .5})
+
+        return img
+
+if __name__ == '__main__':
+    app = MainApp()
+    app.run()
\ No newline at end of file
Index: Mobile App/events.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Mobile App/events.py	(date 1590170914545)
+++ Mobile App/events.py	(date 1590170914545)
@@ -0,0 +1,18 @@
+from kivy.app import App
+from kivy.uix.button import Button
+
+class MainApp(App):
+    def build(self):
+        button = Button(text='Hello from Kivy',
+                        size_hint=(.5, .5),
+                        pos_hint={'center_x': .5, 'center_y': .5})
+        button.bind(on_press=self.on_press_button)
+
+        return button
+
+    def on_press_button(self, instance):
+        print('You pressed the button!')
+
+if __name__ == '__main__':
+    app = MainApp()
+    app.run()
\ No newline at end of file
Index: Mobile App/layouts.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Mobile App/layouts.py	(date 1590170777576)
+++ Mobile App/layouts.py	(date 1590170777576)
@@ -0,0 +1,30 @@
+import kivy
+import random
+
+from kivy.app import App
+from kivy.uix.button import Button
+from kivy.uix.boxlayout import BoxLayout
+
+red = [1, 0, 0, 1]
+green = [0, 1, 0, 1]
+blue = [0, 0, 1, 1]
+purple = [1, 0, 1, 1]
+
+
+class HBoxLayoutExample(App):
+    def build(self):
+        layout = BoxLayout(padding=10)
+        colors = [red, green, blue, purple]
+
+        for i in range(5):
+            btn = Button(text="Button #%s" % (i + 1),
+                         background_color=random.choice(colors)
+                         )
+
+            layout.add_widget(btn)
+        return layout
+
+
+if __name__ == "__main__":
+    app = HBoxLayoutExample()
+    app.run()
